boards/yunjia-nrf51822/Makefile.dep

```diff
@@ -0,0 +1,4 @@
+ifneq (,$(filter ng_netif_default,$(USEMODULE)))
+  USEMODULE += nrf51_ble
+  USEMODULE += ng_nomac
+endif
```

Comment:
If this is needed here, won't it also be needed in makefiles of the other boards? Maybe a global initialization procedure in the [Makefile.dep](https://github.com/RIOT-OS/RIOT/blob/master/Makefile.dep) could be helpful. What do others think?